public-docs/installation-reaction-platform.md

```diff
@@ -0,0 +1,104 @@
+---
+id: installation-reaction-platform
+title: Install with Reaction Platform
+---
+
+Follow these instructions to run the entire Reaction Platform. 
+
+The Platform will install and run the the entire suite of Reaction services in the these directories:
+
+| Directory                                                                                  | Services                        |
+| ------------------------------------------------------------------------------------------ | ------------------------------- |
+| [`reaction`](https://github.com/reactioncommerce/reaction)                                 | GraphQL API, Classic UI, Mongo  |
+| [`reaction-hydra`](https://github.com/reactioncommerce/reaction-hydra)                     | Authentication server, Postgres |
+| [`reaction-next-starterkit`](https://github.com/reactioncommerce/reaction-next-starterkit) | Next.js storefront application  |
+
+## Installation
+
+1. Requirements:
+
+- Install Docker and Docker Compose ([Mac](https://docs.docker.com/docker-for-mac/install/) | [Windows](https://docs.docker.com/docker-for-windows/install/)* | [Linux](https://docs.docker.com/compose/install/#install-compose)**) 
+- Install [Node.js](https://nodejs.org/en/)
+- A GitHub account with a configured SSH key
+
+> *Windows: Reaction Platform has not been fully tested in Windows at this time.
+> **Linux: Docker Compose is included when installing Docker on Mac and Windows, but will need to be installed separately on Linux.
```

Comment:
@rosshadden @focusaurus Does this sound good for Linux users? Are there other things Linux users need to know before they go ahead with this step?